### PR TITLE
Static landing page (WIP)

### PR DESCRIPTION
**Pull request by gdpelican**
I wonder if this works!

The hope to is get our time-to-first-landing down some, as well as provide an entry point for a potential mobile application.

I think this is also a slightly nicer way of managing our third party scripts.

cc @robguthrie @gardner

### CODE REVIEW

File: app/views/angular/boot.js.erb [hunk 1]
```diff
@@ -0,0 +1,21 @@
+var includeJs = function(url, callback) {
```
Comment (gdpelican) [inline app/views/angular/boot.js.erb]: All I have at this point is vanilla javascript and darn does it feel good. #stoneage

File: app/views/angular/boot.js.erb [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+}
+
+var includeCss = function(url) {
+  var elem   = document.createElement('link')
+  elem.href  = url
+  elem.media = 'screen'
+  elem.rel   = 'stylesheet'
+  document.getElementById('boot').appendChild(elem)
+}
+
```
Comment (gdpelican) [inline app/views/angular/boot.js.erb]: I like having these all bunched together like this into a single boot file; makes it really obvious what scripts show up and why

File: app/views/angular/boot/_piwik.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+(function() {
+  var u="//piwik.loomio.org/piwik/";
+  _paq.push(['setTrackerUrl', u+'piwik.php']);
+  _paq.push(['setSiteId', 1]);
+  var d=document, g=d.createElement('script'), s=d.getElementsByTagName('script')[0];
+  g.type='text/javascript'; g.async=true; g.defer=true; g.src=u+'piwik.js'; s.parentNode.insertBefore(g,s);
+})();
+
+// ???
+// <noscript><p><img src="//piwik.loomio.org/piwik/piwik.php?idsite=1" style="border:0;" alt="" /></p></noscript>
```
Comment (gdpelican) [inline app/views/angular/boot/_piwik.js]: Oh right, have to do something about this. What does this do again? Seems mighty fishy to me.

File: app/views/angular/index.html [hunk 4]
```diff
@@ -0,0 +1,20 @@
+<html ng-app="loomioApp" ng-controller="ApplicationController as app">
```
Comment (gdpelican) [inline app/views/angular/index.html]: Beautiful, glorious, regular html, just like mom used to make.

(PS this is the mobile app entry point.)

File: app/views/angular/boot/_piwik.js (hunk 3 above)
Comment (gdpelican) [inline app/views/angular/boot/_piwik.js]: Oh right tracking pixel.

**Comment by gardner** [comment]
This is rad! I'll try to wrap it up in cordova now.

**Comment by gdpelican** [comment]
I think I'm going to try to push this over the line, mostly because I like the idea of grouping our third party dependency scripts together in a really obvious way.

**Comment by gardner** [comment]
Push this pull request over the line?

**Comment by gdpelican** [comment]
Yep, I like the idea of a static page to increase our first-byte time, and then we can do another spike into a cordova thing at some point if we want to.

**Comment by robguthrie** [comment]
Actually.. I'm a little concerned about going "All static" I think we need
to consider a few more things. A dynamic page is only slow if it's doing
lots of work. There are some cases where we do want to some light work and
put it into the boot page, and the time will still be acceptable. My first
thought is facebook/slack previews, do we need to insert meta tags into
this response so they work for discussion, group pages?

**Comment by gdpelican** [comment]
That'd be a thing worth testing.

**Pull request closed without merging**